lib/matplotlib/tests/test_offsetbox.py

```diff
@@ -349,6 +349,11 @@ def test_packers(align):
 
     hpacker = HPacker(children=[r1, r2], pad=0, sep=0, align=align)
     vpacker = VPacker(children=[r1, r2], pad=0, sep=0, align=align)
+
+    # smoke testing default values
```

Comment:
```suggestion
```
Removed the redundant lines.

Also, is not really a smoke test, because the defaults are actually reflected in the image.